src/20_Components/amp-sidebar-1.0.html

```diff
@@ -0,0 +1,147 @@
+<!---
+  {
+    "comment": "amp-sidebar needs to be the first element after the body tag, this is currently not possible the way examples are structured",
+    "experiments": ["amp-sidebar 1.0"],
+    "draft": true,
+    "skipValidation": true
```

Comment:
it might make sense to enable preview mode (this makes it easier to test responsive behavior) by adding:

```
  "preview": "default",
```